FlipperZero/perform_lint.sh

```diff
@@ -0,0 +1,12 @@
+# Lint with systemwide installation if present
+if which swiftlint >/dev/null; then
+  swiftlint
+  exit 0
+fi
+# Download a copy of swiftlint if it doesn't exist in project root
+if [ ! -f swiftlint ]; then
+  curl -o- -Ls https://github.com/realm/SwiftLint/releases/latest/download/portable_swiftlint.zip | tar xzf - swiftlint \
+  && chmod +x swiftlint || echo "warning: Failed to download SwiftLint"; exit 0
+fi
+# Lint with local swiftlint
+./swiftlint
```

Comment:
I'm not sure why, but I don't see any output from the `swiftlint` right after it was downloaded, but it's fine on the next run:
```
$ ./perform_lint.sh
$ ./perform_lint.sh
Linting Swift files at paths 
Linting 'HomeView.swift' (1/4)
Linting 'FlipperZeroApp.swift' (2/4)
Linting 'CoreTests.swift' (3/4)
Linting 'RootView.swift' (4/4)
/.../Flipper-iOS-App/FlipperZero/CoreTests/CoreTests.swift:12:4: warning: Todo Violation: TODOs should be resolved (remove this dummy test when mo...). (todo)
Done linting! Found 1 violation, 0 serious in 4 files.
```
Also, should we add the binary into `.gitignore`?

Reply:
Absolutely it should be ignored. It was already added in #16 
Regarding output, yeah it is also my fault. Gonna fix now.